doc/connectors/google_sheets.md

```diff
@@ -4,20 +4,17 @@
 
 * `type`: `"GoogleSheets"`
 * `name`: str, required
-* `bearer_auth_id`: str, required
+* `auth_id`: str, required
 
-The `bearer_auth_id` is a token retrieved by the front-end
-to let Bearer handle the OAuth dance for you.
+Note: this connector needs a `retrieve_token` function that will be given the `auth_id` and should return a valid token.
```

Comment:
From where is coming this function ?

Reply:
This will be provided by the host (Laputa in our case)